lib/postoffice.ex

```diff
@@ -16,7 +16,7 @@ defmodule Postoffice do
         {:relationship_does_not_exists, %{topic: ["is invalid"]}}
 
       topic ->
-        Messaging.create_message(
+        Messaging.add_message_to_deliver(
```

Comment:
I rename the method because now, we no create a message only, we create a message and a pending message to mark the message this as pending to deliver/publish, I have severals doubts about

`add_message_to_deliver` or `add_message_to_publish`

Rename is very fast, then if you think a better name, tell me please!!!!